source/overview/license-and-subscription.rst

```diff
@@ -109,6 +109,71 @@ Is my license available immediately?
+    "AD/LDAP groups in the database are retained but cannot be used", "- Memberships are frozen in state for group synced teams/channels. 
+    - Mentions for AD/LDAP groups are not shown in the autocomplete menu. 
+    - Group mentions are no longer highlighted in text and do not trigger new notifications.", "Use the `CLI <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group>`_ to modify group sync settings for the team/channel."
+    "High availability is disabled.", "If all nodes in a cluster continue running, the nodes will stop communicating and caches will get out of sync. This is likely to cause delays in messages, notifications, etc.", "None needed."
+    "Performance monitoring is disabled.", "Grafana will no longer update with new data.", "None needed."
+    "Compliance exports jobs are no longer scheduled in the job server", "Data is not exported.", "None needed."
+    "Data retention jobs are no longer scheduled in the job server", "Data is not deleted.", "None needed."
+    "Custom terms no longer show for end users on login", "Data is retained in the Terms of Service database."
+    "Custom announcement banners are no longer visible.", "Default announcement banner displayed.", "None needed."
+    "Multi-factor authentication (MFA)", "MFA is no longer enforced/required for new accounts but remains enabled for those who configured it.", "None needed."
+    "Permissions", "Permissions are retained in the database in a frozen state and cannot be modified in the the System Console.", "Use the `CLI <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-permissions-reset>`_ to reset permissions to default."
+    "Guest accounts", "Guests that are not actively logged in are prevented from logging in. Guests who are actively logged in are able to use Mattermost until their session expires or they log out.", "None needed."
+    
+Why can't I dismiss the expiry notification banner?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If there is a red expiry announcement banner stating: "Enterprise license is expired and some features may be disabled. Please contact your System Administrator for details." it means your grace period has expired. This announcement banner persists until the license is renewed, and is visible to users as well as System Admins.
+
+Once a new license is applied, the banner will no longer be visible. 
+
+If you'd prefer to use Mattermost Team Edition going forward, you need to revoke the expired license by following these steps. 
```

Comment:
Maybe missing a link in "these steps".

Reply:
Good catch, thanks! I had meant to add those.